StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/build/MSBuildOld.props

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (c) Tunnel Vision Laboratories, LLC. All Rights Reserved.
+  Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
❓ If we leave out the `xmlns`, will it still work in Visual Studio 2015?

Reply:
I am not sure, I didn't have a machine with Visual Studio 2015 to test it out.  Its probably safer to have it unless you have a strong opinion to leave it out.